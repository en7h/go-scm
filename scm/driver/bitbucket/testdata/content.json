{
    "mimetype": null,
    "links": {
        "self": {
            "href": "https://api.bitbucket.org/2.0/repositories/tphoney/scm-test/src/0846a192175701903ddd9264ece31922d8437c1a/README.md"
        },
        "meta": {
            "href": "https://api.bitbucket.org/2.0/repositories/tphoney/scm-test/src/0846a192175701903ddd9264ece31922d8437c1a/README.md?format=meta"
        },
        "history": {
            "href": "https://api.bitbucket.org/2.0/repositories/tphoney/scm-test/filehistory/0846a192175701903ddd9264ece31922d8437c1a/README.md"
        }
    },
    "escaped_path": "README.md",
    "path": "README.md",
    "commit": {
        "type": "commit",
        "hash": "0846a192175701903ddd9264ece31922d8437c1a",
        "links": {
            "self": {
                "href": "https://api.bitbucket.org/2.0/repositories/tphoney/scm-test/commit/0846a192175701903ddd9264ece31922d8437c1a"
            },
            "html": {
                "href": "https://bitbucket.org/tphoney/scm-test/commits/0846a192175701903ddd9264ece31922d8437c1a"
            }
        }
    },
    "attributes": [],
    "type": "commit_file",
    "size": 39
}