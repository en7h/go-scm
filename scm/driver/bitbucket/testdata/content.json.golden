{
    "Path": "README",
    "Data": "SEVMTE8gV09STEQK",
    "Sha": "0846a192175701903ddd9264ece31922d8437c1a"
}
